setup.py

```diff
@@ -15,9 +15,24 @@
 import io
 import os
 
-from setuptools import find_packages
-from setuptools import setup
+import setuptools
 
+# Disable version normalization performed by setuptools.setup()
+# Adding this in even though it works for Python3.x, but does not
+# work for Python 2.7
+try:
+    # Try the approach of using sic(), added in setuptools 46.1.0
+    from setuptools import sic
+except ImportError:
+    # Try the approach of replacing packaging.version.Version
+    sic = lambda v: v
+    try:
+        # setuptools >=39.0.0 uses packaging from setuptools.extern
+        from setuptools.extern import packaging
+    except ImportError:
+        # setuptools <39.0.0 uses packaging from pkg_resources.extern
+        from pkg_resources.extern import packaging
+    packaging.version.Version = packaging.version.LegacyVersion
```

Comment:
Instead of the workaround, let's bump the `setuptools` version since it's a dependency.
https://github.com/googleapis/google-auth-library-python/blob/master/setup.py#L29

Reply:
I tried doing that, it works for Python3 but does not work for Python2 :(

Reply:
Ah it must be from setuptools removing support for Python 2. This LGTM.